.gitignore

```diff
@@ -106,3 +106,4 @@ datasets/qm8.csv
 .vscode/
 env.yml
 .python-version
+.spyproject/
```

Comment:
Should this be here?

Reply:
these are specific files of my IDE spyder when I edit modify or debug code. So i think it should be ignored in every commit .